storage/cloud-client/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-storage</artifactId>
-      <version>0.6.0</version>
+      <version>0.7.0</version>
```

Comment:
Odd that this isn't 0.8.0.

Ah, I see that it was tagged beta. http://search.maven.org/#artifactdetails%7Ccom.google.cloud%7Cgoogle-cloud-storage%7C0.8.0-beta%7Cjar